docs/source/admin/ansible-labs/ansible_labs.rst

```diff
@@ -0,0 +1,172 @@
+-----------------
+
+A simple scaffold for a lab directory is included at :atc-file:`infrastructure/ansible/sample.lab`.  This directory should encapsulate all pieces that make one environment unique from another.  Ideally making new environments is as simple as copy-paste this directory and tweak the variables desired inside.
+
+* The :atc-file:`infrastructure/ansible/sample.lab/ansible` subdirectory should be used to hold variables specific to a particular lab in either ``vars.yml`` or an encrypted Ansible ``vault``
+* The :atc-file:`infrastructure/ansible/sample.lab/inventory` directory is where it's recommended for your provisioning layer to drop a valid Ansible inventory file describing what was allocated.  When using Ansible, it's important to point the inventory source to this directory so that it will merge all available inventory files together for you.
+* The :atc-file:`infrastructure/ansible/sample.lab/out/ssl` directory is generated with the first run of the lab and holds your local copy of the lab SSL data.  The out directory is also handy for holding temporary data from the provisioning or steady-state layers to help triage failures.
+* The docker and docker-compose related files are present as an optional wrapper for Linux hosts (doesn't work on OSX) around all the lab plumbing dependencies for Ansible.  This is particularly handy for automated systems who perform regular redeployments such as in a CI/CD tool.
+* :atc-file:`infrastructure/ansible/sample.lab/manual.run.sh` is a scaffold for the entrypoint for performing a lab rebuild from your local system.
+
+Gilt
+""""
+
+Traditionally when distributing application playbooks for Ansible, many people use the built-in Ansible Galaxy repository.
+There is a design limitation to the Ansible Galaxy though in that one git repository may only contain one role.
+In the case of Apache Traffic Control, there are many components each with their own roles.
+At the end of the day, the generic core roles must exist in a valid Ansible role directory location.
+There are many solutions to this problem, but one of the better and easier once that's been run across is using the 3\ :sup:`rd`-party tool `Gilt <https://github.com/metacloud/gilt>`_.
+As another alternative you can simply extract the roles from an Apache Traffic Control (ATC) source tarball from a build.
+
+The Roles directory
+-------------------
+
+The generic core roles for each component live at :atc-file:`infrastructure/ansible/roles`.
+Each role contains a README.md with more information, but this is not a replacement for existing documentation on the components themselves.
+It's very useful to still review the Administrator's Guide in the documentation as you develop your implementation around the component's generic core.
+
+If you're attempting to optimize the wallclock time needed to deploy all the components in parallel, they should be installed like the following:
+
+.. figure:: ATC.installation.dependencies.svg
```

Comment:
The build can't seem to find the file `ATC.installation.dependencies.svg` because there is a case issue with the file name.  The `installation` part of the file name begins with a capital `I` instead of lowercase as specified on line 152 above.

The image files I show in the `ansible-labs` documentation directory under `admin`: 
```
$ find .. -name ATC*
../admin/ansible-labs/ATC.Installation.dependencies.svg  <-------- Mixed case
../admin/ansible-labs/ATC.lab.layers.svg
```

Warning from the build about the missing image file:
```
/src/trafficcontrol/docs/source/admin/ansible-labs/ansible_labs.rst:: WARNING: image file not readable: admin/ansible-labs/ATC.installation.dependencies.svg
```

Reply:
Addressed with commit.